libllvm build problem - omp.h No such file or directory

**Problem description**
I'm trying to build libllvm using the termux fork and termux packages workflow
the build step failed with error:
```
cp: cannot stat '../src/projects/openmp/runtime/exports/common.min/include/omp.h': No such file or directory
##[error]Process completed with exit code 1.
```

Could you please point me to the right direction how to solve this problem? Thanks in advance.

**Steps to reproduce**
1. fork termux
2. add action "Packages"
3. bump libllvm revision to start the build action 

**Expected behavior**
build step passed

**Additional information**
you can see this here: https://github.com/araczkowski/termux-packages/runs/735649262?check_suite_focus=true

![image](https://user-images.githubusercontent.com/1263968/83685404-e7e8a280-a5e8-11ea-99b0-cd59646e8948.png)

--- a/packages/libllvm/build.sh
+++ b/packages/libllvm/build.sh
@@ -102,9 +102,9 @@ termux_step_pre_configure() {
 }
 termux_step_post_make_install() {
 	if [ $TERMUX_ARCH = "arm" ]; then
-		cp ../src/projects/openmp/runtime/exports/common.min/include/omp.h $TERMUX_PREFIX/include
+		cp $TERMUX_PKG_SRCDIR/projects/openmp/runtime/exports/common/include/omp.h $TERMUX_PREFIX/include
 	else
-		cp ../src/projects/openmp/runtime/exports/common.min.ompt.optional/include/omp.h $TERMUX_PREFIX/include
+		cp $TERMUX_PKG_SRCDIR/projects/openmp/runtime/exports/common.ompt.optional/include/omp.h $TERMUX_PREFIX/include
 	fi	
 	if [ "$TERMUX_CMAKE_BUILD" = Ninja ]; then
 		ninja docs-llvm-man

